### ISSUE TITLE
Optional feature for embedding Last.fm

### ISSUE THREAD

**Issue by PxlBuzzard** [jakke-korpelainen/newtab#24]
In a similar vein to #12, it would be cool and relatively simple to hook into Last.fm's [currently playing API](http://www.last.fm/api/show/user.getRecentTracks). You don't need auth or anything, just the user's Last.fm username. This would show the currently playing or last-listened-to track.

**Comment by jakke-korpelainen** [comment]
Yeah this would be nice. Need to think how to make the base for these optional features, would be cool if the user also could choose the position where to display each field (drag&drop).
